scripts/cpu-stats.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+#
+# Reports cpu usage statistics
+#
+set -e
+
+[[ $(uname) == Linux ]] || exit 0
+
+cd "$(dirname "$0")"
+
+# shellcheck source=scripts/configure-metrics.sh
+source configure-metrics.sh
+
+cpu_usage=0
+
+update_cpustat() {
+# collect the total cpu usage by subtracting idle usage from 100%
+cpu_usage=$(top -bn1 | grep '%Cpu(s):' | sed "s/.*, *\([0-9.]*\)%* id.*/\1/" | awk '{print 100 - $1}')
```

Comment:
nit: indent by 2

Reply:
actually `update_cpustat()` can just be inlined in the main while loop it looks like

Reply:
I left it like this so I can follow up with RAM and possible GPU stats

Reply:
ah, ok.  Then remove the extra `update_cpustat` call on line 21?

Reply:
That initializes it so it doesn't end up sending 0 the first time?